app/jobs/upload_uphold_access_parameters_job.rb

```diff
@@ -0,0 +1,11 @@
+class UploadUpholdAccessParametersJob < ApplicationJob
```

Comment:
@bsclifton the Sidekiq web interface is mounted at `/magic` by default in [routes.rb](https://github.com/brave/publishers/blob/master/config/routes.rb#L67)